### DISCUSSION TITLE
Security focus question

### DISCUSSION THREAD

**Discussion by Koliera** [tteck/Proxmox#477] [Ideas]
Love your scripts and am happy to see that there are continuous improvements to the scripts. 
One question I had was whether there was something that could be done to have some security focus for the containers? 
One example of interest I've seen is a script that installs Wireguard with additional hardening (Authelia, Bunker, etc):
https://github.com/notthebee/ansible-easy-vpn
https://www.youtube.com/watch?v=SMF301vQqJo

Any ideas of what could be done to harden ProxMox itself or the containers within?

**Comment by tteck** [comment]
Unprivileged containers are safe by design. 
But, they're a few scripts in the repo you may consider, such as CrowdSec, Keycloak and Tailscale
